### ISSUE TITLE
Could we support lower dynamic-shape onnx to dynamic-shape MHLO?

### ISSUE THREAD

**Issue by hijack4312** [onnx/onnx-mlir#2144]
when I tried to transfer an ONNX model to mlir dialect using --EmitMLIR, I found the dialect is based on 'arith.op', could we support to transfer the onnx to 'mhlo.op'?

**Comment by AlexandreEichenberger** [comment]
Based on an earlier slack exchange, I believe @hijack4312 is asking for an `--EmitMHLO` option that lower onnx to HMLO.

**Comment by hijack4312** [comment]
Hi, Alex, that is right. Thanks for explanation.

**Comment by chenchongsong** [comment]
> Based on an earlier slack exchange, I believe @hijack4312 is asking for an `--EmitMHLO` option that lower onnx to HMLO.
> 

Yes, will implement this option in the future PR! cc @yaochengji

**Comment by yaochengji** [comment]
At present, the  onnx-model-to-mhlo conversion pipeline is managed within th [byteir's onnx frontend](https://github.com/bytedance/byteir/blob/main/frontends/onnx-frontend/onnx-frontend/src/Compiler/OFCompilerPipelines.cpp), which includes some customized configurations.. @hijack4312 please feel free to utilize this resource if it meets your requirements. Additionally, we are open to incorporating a streamlined onnx-model-to-mhlo pipeline within the `onnx-mlir` repository, @AlexandreEichenberger .
